features/step_definitions/feedback_steps.rb

```diff
@@ -47,12 +20,16 @@
   end
 end
 
+Then('I see a warning that my feedback was not submitted successfully') do
+  expect(page).to have_content "Unable to submit feedback [1050]"
```

Comment:
can this be loosened to use a regex for the `id` (i assume that is some sort of id).

```suggestion
  expect(page).to have_content(/Unable to submit feedback \[\d+\]/)
```

Reply:
**nit**: i see it is a specific error code that provides us with info on the error, but:
a.  it might be worth loosening anyway
b. perhaps we should not bother showing this to the user anyway?

Reply:
The code can be one of several values and we are showing it to the user so that, if we get an error report, we can more easily find the cause. See https://developer.surveymonkey.com/api/v3/#error-codes